src/houston/public/styles/houston.css

```diff
@@ -8,37 +8,28 @@
 ******/
 
 nav {
-    background-color: #3a416f;
+    background-color: #403757;
```

Comment:
You've changed `src/houston/public/styles/index.css` to transparent, but not `src/houston/public/styles/houston.css`

Is that intentional?